### PR TITLE
✨ Add function to update existing password

### PR DESCRIPTION
**Pull request by phlgr**

### CODE REVIEW

**Comment by marwinburesch** [comment]
That's great mate I'd rate it 8 out of 8!

**Comment by phlgr** [comment]
> That's great mate I'd rate it 8 out of 8!

Thank you Marwin very cool!

File: app.js [hunk 1]
```diff
@@ -27,13 +27,20 @@ async function validateAccess() {
...
-  const password = passwordSafe[passwordName];
-  if (password) {
-    console.log(`Password is ${password}`);
+  if (newPasswordValue) {
+    passwordSafe[passwordName] = newPasswordValue;
+    fs.writeFile("./db.json", JSON.stringify(passwordSafe, null, 2));
   } else {
-    console.log("Unknown password");
+    console.log(`You want to know the password of '${passwordName}'`);
+
```
Comment (lmachens) [inline app.js:37]: 🍕

Comment (lmachens) [reply app.js:37]: https://github.com/lmachens/pw4u/pull/5/files#r526778047

**Pull request merged**